src/relcast.erl

```diff
@@ -620,8 +623,17 @@ stop(Reason, State = #state{module=Module, module_state=ModuleState})->
             ok
     end,
     State1 = maybe_serialize(State),
-    catch rocksdb:transaction_commit(State1#state.transaction),
-    rocksdb:close(State#state.db).
+    %% catch rocksdb:transaction_commit(State1#state.transaction),
```

Comment:
Why was this commented out?

Reply:
there were two paths here.  I wasn't sure which one was being used, or if we were using both.  So I simplified to this, which seems to work more consistently.  I need to do more thinking about this particular code path before we merge any of the changes other than the create bit above.